packages/foundation/src/themeProvider.tsx

```diff
@@ -0,0 +1,51 @@
+import * as React from 'react';
+
+export interface IThemeProviderProps<TScheme> {
+  scheme: TScheme;
```

Comment:
would this also include theme?

Reply:
Well, I didn't add it here since `Customizer` handles that. We could add it here as a convenience but I didn't want to duplicate Customizer function off the bat.